packages/dd-trace/test/plugins/agent.js

```diff
@@ -35,59 +36,66 @@ function ciVisRequestHandler (request, response) {
   })
 }
 
-function addEnvironmentVariablesToHeaders (headers) {
-  // get all environment variables that start with "DD_"
-  const ddEnvVars = Object.entries(process.env)
-    .filter(([key]) => key.startsWith('DD_'))
-    .map(([key, value]) => `${key}=${value}`)
-
-  // add the DD environment variables to the header if any exist
-  // to send with trace to final agent destination
-  if (ddEnvVars.length > 0) {
-    headers['X-Datadog-Trace-Env-Variables'] = ddEnvVars.join(',')
-  }
-}
-
-function handleTraceRequest (req, res, sendToTestAgent) {
+async function handleTraceRequest (req, res, sendToTestAgent) {
+  res.status(200).send({ rate_by_service: { 'service:,env:': 1 } })
   // handles the received trace request and sends trace to Test Agent if bool enabled.
   if (sendToTestAgent) {
     const testAgentUrl = process.env.DD_TEST_AGENT_URL || 'http://127.0.0.1:9126'
+    let schemaVersion
+    let expectedServiceName
+    let traceEnvHeader = ''
```

Comment:
Why is this needed? If we need to set it manually, then we're not really testing a real scenario.